hack/push-container-manifest.sh

```diff
@@ -34,6 +34,9 @@ function podman_push_manifest() {
     ${KUBEVIRT_CRI} manifest create ${DOCKER_PREFIX}/${image}:${DOCKER_TAG}
     for ARCH in ${BUILD_ARCH//,/ }; do
         FORMATED_ARCH=$(format_archname ${ARCH})
+        if [ ! -f ${DIGESTS_DIR}/${FORMATED_ARCH}/bazel-bin/push-$image.digest ]; then
```

Comment:
Shall we apply the same logic on the function `docker_push_manifest`?

Reply:
Yes, absolutely. I just don't use docker, so i forgot. I will add it there as well.